floor_generator/lib/writer/query_method_writer.dart

```diff
@@ -192,6 +203,30 @@ class QueryMethodWriter implements Writer {
 
     return 'return _queryAdapter.query$list$stream($parameters);';
   }
+
+  String _generateDartCoreMapper(final DartType returnType) {
+    final castedDatabaseValue = 'row.values.first'.cast(
+      returnType,
+      returnType.element,
+      withNullability: false,
+    );
+    return '(Map<String, Object?> row) => $castedDatabaseValue';
+  }
+
+  String _generateConverterMapper(final TypeConverter typeConverter) {
+    final castedDatabaseValue = 'row.values.first'.cast(
+      typeConverter.databaseType,
+      typeConverter.fieldType.element,
+    );
+    return '(Map<String, Object?> row) => _${typeConverter.name.decapitalize()}.decode($castedDatabaseValue)';
+  }
+
+  String _parseTableName(String query) {
+    return RegExp(r'(?<=FROM )\w+', caseSensitive: false)
+            .firstMatch(query)
+            ?.group(0) ??
+        'no_table_name';
```

Comment:
You might want to have this next to the ??
Just personal preference tho.

Reply:
yes, it would be nice, but `flutter format` has a different opinion 😁 
I think we need to keep the standard code formatting configuration. Otherwise, our pipeline will be broken.